crates/kitsune2/tests/blocks.rs

```diff
@@ -147,6 +150,39 @@ impl TxHandler for TestTxHandler {
     }
 }
 
```

Comment:
Runs an sbd server and creates a builder using the tx5 transport factory and with the appropriate signal server url. It returns the sbd server instance as well, despite never explicitly being used, since dropping it would stop the server.